components/builder-depot/src/server.rs

```diff
@@ -2549,7 +2569,7 @@ mod test {
             },
             "checksum":"checksum",
             "manifest":"manifest",
-            "target":"x86_64-linux",
+            "platform":"x86_64-linux",
```

Comment:
We seem to be using the terminology for Target to mean Platform + Architecture... do we want to be changing that here?

Reply:
I was just trying to be consistent, but I see your point.  I'll change it back.